### ISSUE TITLE
besside-ng-crawler fail destination path of CapFileOut

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#817]
**Reported by Jano on 22 Nov 2010 12:57 UTC**

-If you specify for CapFileOut a target patch/directory different from /home, the script goes in loop to infinity, and in a few seconds it generating a data file of several Gb.

For reproduce:

```
besside-ng-crawler ~/Scrivania/test/ ~/Scrivania/test/wpa_crawler.cap
``` 

- I suggest you activate a feature that saves the CapFileOut correctly in different patches. (If this information is indicated)

Jano

**Comment by aircrack-ng** [comment]
**Comment by ASPj on 22 Nov 2010 12:57 UTC**

yeah, i know, it crawls its own output file.

we need to check if the output file is in the crawler directory and drop an error in this case.

**Comment by aircrack-ng** [comment]
**Comment by Jano on 22 Nov 2010 12:57 UTC**

Hi.., now I understand, but then this indication is best to write / indicate, because if someone points to the same destination fills the HDD in few minutes, and is not nice if it happens.

Jano

**Comment by aircrack-ng** [comment]
**Comment by ASPj on 22 Nov 2010 12:57 UTC**

fixed in Changeset #1817 :)
